en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-graphql-api-in-choreo-connect.md

```diff
@@ -0,0 +1,296 @@
+
+    ``` 
+    query{
+    human(id:1000){
+        id
+        name
+    }
+    droid(id:2000){
+        name
+        friends{
+            name
+            appearsIn
+        }
+    }
+    }
+    ```
+
+2. Click **Execute**.
+
+    You will get a response for the GraphQL API invocation, as shown in below.
+
+    [![Response of GraphQL Query]({{base_path}}/assets/img/learn/graphql-response-query.png)]({{base_path}}/assets/img/learn/graphql-response-query.png)
+
+## Via apictl for Standalone Mode
+
+The CLI tool ([**apictl**]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl)) 
+does not support initializing projects for GraphQL APIs yet. However, you can download the sample GraphQL API project in 
+[here](https://github.com/wso2/product-microgateway/tree/main/samples/apiProjects/SampleGraphQLApi) and try out it by deploying 
+to the Choreo Connect. When you are using your own GraphQL API with Choreo Connect, you can change the relevant attribute values 
+in `api.yaml`, `graphql-complexity.yaml` and `definitions/schema.graphql` files. There is an explanation regarding those attribute values in 
+[here](https://github.com/wso2/product-microgateway/tree/main/samples/apiProjects/apiProjects/SampleGraphQLApi/README.md).
+
+The following steps describe how to deploy GraphQL API in Choreo Connect standalone mode.
+
+### Step 1 - Create an API Project
+
+When using GraphQL APIs in Choreo Connect standalone mode, you need to have an API project. Structure of a sample GraphQL API project included in below.
+
+```bash
+.
+├── Definitions
+│   ├── graphql-complexity.yaml
+│   └── schema.graphql
+└── api.yaml
+
+```
+
+!!! note
+    - Similar to the sample project, you can deploy your own GraphQL API in Choreo Connect by changing the relevant attribute values that are defined in the `api.yaml` file.
+    - `graphql-complexity.yaml` is an optional file where you can define complexities relevant to the GraphQL queries.
+
+### Step 2 - Add a Choreo Connect Environment to apictl
+
+To use apictl with Choreo Connect, a Choreo Connect environment needs to be added to the apictl. This environment will hold the adapter URL for further commands.
+
+``` bash
+apictl mg add env dev --adapter https://localhost:9843
+```
+
+!!! info
+
+    Note `mg` in the above command. The apictl commands that starts as `apictl mg` are Choreo Connect specific. If a command does not have `mg` after `apictl` then the command could probably be common to both Choreo Connect and API Manager, but it could also be API Manager specific. 
```

Comment:
```suggestion
    The apictl commands that starts as `apictl mg` are Choreo Connect specific. If a command does not have `mg` after `apictl` then the command could probably be common to both Choreo Connect and API Manager, but it could also be API Manager specific. 
```

Reply:
Removed an unnecessary sentence here.